src/opm/input/eclipse/Schedule/Well/Connection.cpp

```diff
@@ -29,7 +29,6 @@
 #include <opm/input/eclipse/Deck/DeckKeyword.hpp>
 #include <opm/input/eclipse/Deck/DeckRecord.hpp>
 #include <opm/input/eclipse/EclipseState/Grid/FieldPropsManager.hpp>
-#include <opm/input/eclipse/Schedule/Well/Connection.hpp>
```

Comment:
We generally try to have all `.cpp` files include their own `.hpp` file.  Is there a reason for not doing that here?